scalameta/io/shared/src/main/scala/scala/meta/io/RelativePath.scala

```diff
@@ -6,12 +6,25 @@ import java.nio.file.Path
 import java.nio.{file => nio}
 import java.nio.file.Paths
 import scala.meta.internal.io.PathIO
+import scala.collection.JavaConverters._
 
 /** Wrapper around a relative nio.Path. */
 sealed abstract case class RelativePath(toNIO: Path) {
   require(!toNIO.isAbsolute, s"$toNIO is not relative!")
   def toFile: File = toNIO.toFile
-  def toURI: URI = toFile.toURI
+  def toURI: URI = {
```

Comment:
If not, I propose that we drop the no-parameter overload, since it seems that `isDirectory` is a core part of the API that doesn't have a universally useful default value.